webview_panels/src/modules/defer/DeferToProduction.tsx

```diff
@@ -93,23 +125,102 @@ const DeferToProduction = (): JSX.Element => {
     }
   };
 
-  const handleManifestPathChange = (
+  const handleLocalManifestPathChange = (
     event: React.ChangeEvent<HTMLInputElement>,
   ) => {
     const { value, name } = event.target;
+    if (!value.endsWith("manifest.json")) {
+      setShowManifestError(false);
+    } else {
+      setShowManifestError(true);
+    }
     setDeferState((prevState) => ({ ...prevState, [name]: value }));
   };
 
-  const onManifestBlur = async () => {
-    const response = await executeRequestInSync("updateDeferConfig", {
-      key: "manifestPathForDeferral",
-      value: manifestPathForDeferral,
-      isPreviewFeature: true,
+  const onLocalManifestBlur = async () => {
+    if (!showManifestError) {
+      const response = await executeRequestInSync("updateDeferConfig", {
```

Comment:
you could modify `updateDeferConfig` to accept of of key value pairs to avoid multiplecalls